packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol

```diff
@@ -86,6 +82,18 @@ abstract contract CrossDomainMessenger is
      */
     address public immutable OTHER_MESSENGER;
 
+    /**
+     * @custom:spacer ReentrancyGuardUpgradeable
+     * @notice Spacer for backwards compatibility
+     */
+    uint256 private _status;
```

Comment:
Small nit here, this should follow the spacer syntax and an entry to the storage lockfile should be added. Happy to open a PR to this PR implementing this if unclear how to do this

Reply:
Done :+1: